### PR TITLE
OOP school library: Add basic UI

### PR DESCRIPTION
**Pull request by amrendrakind**
### Add basic UI
- Watch the video with the UI example again (you have seen it while reading the Sneak Peek).
[UI example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
image
- Console app should behave in the same way is in the example.
- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
   - List all books.
   - List all people.
   - Create a person (teacher or student, not a plain `Person`).
   - Create a book.
   - Create a rental.
   - List all rentals for a given person id.

- In your `main.rb` define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
   - Present the user with a list of options to perform.
   - Lets users choose an option.
   - If needed, ask for parameters for the option.
   - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    @books.each_with_index { |book, index| puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}" }
+    book_number = gets.chomp.to_i
+    puts 'Select a Person from the following list by number'
+    @persons.each_with_index do |person, index|
+      puts " #{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    person_number = gets.chomp.to_i
+    print 'Date: '
+    date = gets.chomp
+    @rentals = Rental.new(date, @books[book_number], @persons[person_number])
```
Comment (AnselemOdims) [inline app.rb]: - [ ] You are supposed to be pushing the newly created rental into the array and not assigning it directly. Assigning it directly means that rentals is not an array but just an instance of the rental class. This makes it throw an error when you try to list all rentals for a user.
Please kindly push every new rental into the rentals array

```
@rentals << Rental.new(date, @books[book_number], @persons[person_number])
```

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  end
+
+  def list_rentals
+    puts 'List all rentals'
+    print 'Enter ID of person: '
+    person_id = gets.chomp
+    puts 'Rentals : '
+    @rentals.each do |rent|
+      if rent.person.id == person_id
+        puts "#{rent.class} #{rent.date} | Book: \"#{rent.book.title}\" rented by #{rent.person.name}"
```
Comment (AnselemOdims) [inline app.rb]: - [ ] As pointed out above, because `@rentals` is not an array then `.each` is going to fail. Please correct the above as pointed out.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @amrendrakind , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 1 above)
Comment (amrendrakind) [reply app.rb]: @AnselemOdims Thank you for your feedback.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @amrendrakind, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Descriptive PR title and description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**